README.md

```diff
@@ -1,6 +1,6 @@
 # Hack Club Global Websites
 
-The goal of this project is to enable the creation of simplified versions of our Hack Club site that are translated into global languages. These sites should require practically no maintenance and be very brief. An example is [`hackclub.es`](https://hackclub.es).
+The goal of this project is to enable the creation of simplified versions of our Hack Club site that are translated into global languages. These sites should require practically no maintenance and be very brief. An example is [`hackclub.es`](https://gr.hackclub.com).
```

Comment:
Hi! Can you change the link display to `gr.hackclub.com` as to much the link?

Reply:
Oh right my bad ! I'll do that.